apps/activity/views.py

```diff
@@ -1,5 +1,179 @@
+        serializer.is_valid(raise_exception=True)
+        total_calories = serializer.data['all_calories']
+        response_data = {
+            'all_calories': total_calories,
+            'foods': foods_serializer.data
+        }
+        return Response(response_data, status=status.HTTP_200_OK)
+
+    @action(methods=['post'], detail=False, serializer_class=CreateStepsSerializer, url_path="step")
+    def create_step(self, request, *args, **kwargs):
+        plan = self.request.user.plan
+        serializer = self.get_serializer(data=request.data)
+        serializer.is_valid(raise_exception=True)
+        serializer.save(plan=plan)
+        return Response({"success": True}, status.HTTP_201_CREATED)
+
+    @action(methods=['get'], detail=False, serializer_class=GetAllStepsSerializer, url_path="steps")
+    def steps(self, request, *args, **kwargs):
+        steps = ActivityStep.objects.filter(plan=self.request.user.plan, created_at__date=timezone.now()).order_by("-id")
+        steps_serializer = CreateStepsSerializer(steps, many=True)
+        serializer = self.get_serializer(data=request.data)
+        serializer.is_valid(raise_exception=True)
+        total_steps = serializer.data['all_steps']
+        response_data = {
+            'all_steps': total_steps,
+            'steps': steps_serializer.data
+        }
+        return Response(response_data, status=status.HTTP_200_OK)
+
+    @action(methods=['get'], detail=False, serializer_class=GetTaskSerializer, url_path="tasks")
+    def my_tasks(self, request, *args, **kwargs):
+        return super().list(request, *args, **kwargs)
+
+    @action(methods=['get'], detail=True, serializer_class=None, url_path="add-to-tasks")
+    def add_to_my_tasks(self, *args, **kwargs):
+        task = get_object_or_404(Task, pk=kwargs.get("pk"))
+        plan = self.request.user.plan
+        plan.tasks.add(task)
+        plan.save()
+        return Response({"success": True}, status.HTTP_200_OK)
+
+    @action(methods=['get'], detail=True, serializer_class=None, url_path="delete-from-tasks")
+    def delete_from_my_tasks(self, *args, **kwargs):
+        task = get_object_or_404(Task, pk=kwargs.get("pk"))
+        plan = self.request.user.plan
+        plan.tasks.remove(task)
+        plan.save()
+        return Response({"success": True}, status.HTTP_200_OK)
+
+    @action(methods=['get'], detail=True, serializer_class=None, url_path="start-task")
+    def start_task(self, *args, **kwargs):
+        task = Task.objects.get(id=kwargs.get("pk"))
+        plan = self.request.user.plan
+        task_exists = plan.tasks.filter(id=task.id).exists()
+        if task_exists:
+            plan.started_task = task
+            plan.start_task = timezone.now()
+            plan.save()
+            return Response({"success": True}, status.HTTP_200_OK)
+        return Response({"success": False}, status.HTTP_200_OK)
+
+    @action(methods=['get'], detail=False, serializer_class=None, url_path="continue-task")
```

Comment:
Nu sunt exact cum arata acuma in swagger body la endpoint dar deobicei daca nu dorim body folosim decoratorul 
@swagger_auto_schema(request_body=no_body) 
P.S te rog sa verifici toate endpointurile unde este problema data.